packages/modal/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "pcln-modal",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Comment:
Version updates can wait until the publishing process, as per the [Publishing guidelines](https://github.com/pricelinelabs/design-system/blob/master/CONTRIBUTING.md#publishing)